### ISSUE TITLE
testsuite missing examples/ files

### ISSUE THREAD

**Issue by juhp** [haskell-tools/haskell-tools#223]
```
Linking dist/build/haskell-tools-test/haskell-tools-test ...
> /tmp/stackage-build12/haskell-tools-refactor-0.4.1.0$ dist/build/haskell-tools-test/haskell-tools-test
all tests
  functional tests
    reprint tests
      Decl.AmbiguousFields:                                          FAIL
        Exception: ../../examples/Decl/AmbiguousFields.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.AnnPragma:                                                FAIL
        Exception: ../../examples/Decl/AnnPragma.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.ClosedTypeFamily:                                         FAIL
        Exception: ../../examples/Decl/ClosedTypeFamily.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.CtorOp:                                                   FAIL
        Exception: ../../examples/Decl/CtorOp.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.DataFamily:                                               FAIL
        Exception: ../../examples/Decl/DataFamily.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.DataType:                                                 FAIL
        Exception: ../../examples/Decl/DataType.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.DataTypeDerivings:                                        FAIL
        Exception: ../../examples/Decl/DataTypeDerivings.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.FunBind:                                                  FAIL
        Exception: ../../examples/Decl/FunBind.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.FunctionalDeps:                                           FAIL
        Exception: ../../examples/Decl/FunctionalDeps.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.FunGuards:                                                FAIL
        Exception: ../../examples/Decl/FunGuards.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.GADT:                                                     FAIL
        Exception: ../../examples/Decl/GADT.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.InjectiveTypeFamily:                                      FAIL
        Exception: ../../examples/Decl/InjectiveTypeFamily.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.InlinePragma:                                             FAIL
        Exception: ../../examples/Decl/InlinePragma.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.InstanceOverlaps:                                         FAIL
        Exception: ../../examples/Decl/InstanceOverlaps.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.InstanceSpec:                                             FAIL
        Exception: ../../examples/Decl/InstanceSpec.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.LocalBindings:                                            FAIL
        Exception: ../../examples/Decl/LocalBindings.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.LocalFixity:                                              FAIL
        Exception: ../../examples/Decl/LocalFixity.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.MultipleFixity:                                           FAIL
       Exception: ../../examples/Decl/MultipleFixity.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.MultipleSigs:                                             FAIL
        Exception: ../../examples/Decl/MultipleSigs.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.OperatorBind:                                             FAIL
        Exception: ../../examples/Decl/OperatorBind.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.OperatorDecl:                                             FAIL
        Exception: ../../examples/Decl/OperatorDecl.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.ParamDataType:                                            FAIL
        Exception: ../../examples/Decl/ParamDataType.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.PatternBind:                                              FAIL
        Exception: ../../examples/Decl/PatternBind.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.PatternSynonym:                                           FAIL
        Exception: ../../examples/Decl/PatternSynonym.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.RecordPatternSynonyms:                                    FAIL
        Exception: ../../examples/Decl/RecordPatternSynonyms.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.RecordType:                                               FAIL
        Exception: ../../examples/Decl/RecordType.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.RewriteRule:                                              FAIL
        Exception: ../../examples/Decl/RewriteRule.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.SpecializePragma:                                         FAIL
        Exception: ../../examples/Decl/SpecializePragma.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.StandaloneDeriving:                                       FAIL
        Exception: ../../examples/Decl/StandaloneDeriving.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.TypeClass:                                                FAIL
        Exception: ../../examples/Decl/TypeClass.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.TypeClassMinimal:                                         FAIL
        Exception: ../../examples/Decl/TypeClassMinimal.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.TypeFamily:                                               FAIL
        Exception: ../../examples/Decl/TypeFamily.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.TypeFamilyKindSig:                                        FAIL
        Exception: ../../examples/Decl/TypeFamilyKindSig.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.TypeInstance:                                             FAIL
        Exception: ../../examples/Decl/TypeInstance.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.TypeRole:                                                 FAIL
        Exception: ../../examples/Decl/TypeRole.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.TypeSynonym:                                              FAIL
        Exception: ../../examples/Decl/TypeSynonym.hs: openBinaryFile: does not exist (No such file or directory)
      Decl.ValBind:                                                  FAIL
        Exception: ../../examples/Decl/ValBind.hs: openBinaryFile: does not exist (No such file or directory)
      Expr.ArrowNotation:                                            FAIL
:
:
:
```

**Comment by nboldi** [comment]
This problem should be solved by #226. A new bugfix version is out: haskell-tools-*:0.4.1.1

**Comment by juhp** [comment]
(Sorry typo, that should read "re-enable rest of...")
